AmethystLauncher/src/loader/Loader.cpp

```diff
@@ -109,11 +109,21 @@ void ModLoader::InjectRuntime() {
         mod_shortened = mod_shortened.substr(0, atPos);
     }
 
-    if (!fs::exists(mModsPath)) {
+    std::string oldModPath = mAmethystPath + "/mods";
+    bool oldModDirExists = fs::exists(oldModPath);
+    bool modDirExists = fs::exists(mModsPath);
+
+    if(!modDirExists && oldModDirExists) {
+        Log::Warning("Mods detected at the old directory: {0}\nCopying them to the new location: {1}", oldModPath, mModsPath);
```

Comment:
We should avoid having backwards compatibility features for now TBH, while the amount of people using the project is small